### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by gregoryjscott**
Remaining Tasks
- [x] Wait for #20 to be merged
- [ ] Tidy up the carriage returns 
- [ ] Squash to one commit after changes are complete

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# Contributing
+
+Welcome! This document is intended to turn you into a Simpler contributor. The
+goal is to make that process as smooth and painless as possible. Please
+follow the instructions below to get setup.
```
Comment (kamillf) [inline CONTRIBUTING.md:5]: Setup should be to words in this instance: "set up"

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+If you experience problems or have feedback on this document then please
```
Comment (kamillf) [inline CONTRIBUTING.md:7]: Add a comma after _document_ and remove _then_

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+read the section on [Issues](#issues) and let us know about it.
+
+## Git and GitHub
+
+We use git and GitHub to store and manage our content. Content is not just
+source code - it's code, documentation, scripts, notes, etc.
+
+### Steps
+
+1. Setup a [GitHub](https://github.com) account.
```
Comment (kamillf) [inline CONTRIBUTING.md:17]: Setup should be two words here as well (the verb form is 2 words; the noun form is 1 word)

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+1. [Install git](https://help.github.com/articles/set-up-git).
+
+>Technically, simple contributions (e.g. fixing a typo in a document) can
```
Comment (kamillf) [inline CONTRIBUTING.md:20]: Should be a comma after _e.g._

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+However, this document assumes the user is using git and the user's local
+drive to push contributions to GitHub.
+
+## Issues
+
+We use GitHub [Issues](https://guides.github.com/features/issues/) to track the
+backlog of work including bugs. Before submitting an issue, please browse the
+[current list](https://github.com/gregoryjscott/Simpler/issues) to see if your
+issue has already been identified. If it isn't clear if your issue is
+unique then go ahead and enter a new issue. Thanks in advance for your
```
Comment (kamillf) [inline CONTRIBUTING.md:31]: I would remove _then go ahead and_ and insert a comma after _unique_

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+The `master` branch should be considered the "edge" branch and should be the basis for all Pull
+Requests. Other important branches should be associated to an open Pull Request.
+
+There are [tags](https://github.com/gregoryjscott/Simpler/tags) for each Simpler release. If you
+are building Simpler for production use then you'd be wise to use the latest tagged version of the source.
+
+## Continuous Integration
+
+We believe strongly in the benefits or [Continuous Integration]
```
Comment (kamillf) [inline CONTRIBUTING.md:72]: Typo--_or_ should be _of_

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+(http://en.wikipedia.org/wiki/Continuous_integration) (CI). All the steps to
+achieve CI must be automated in order for CI to be successful. We use
+[rake](http://rake.rubyforge.org/) tasks to drive the automation. You'll need
+ruby and bundler installed to run rake tasks.
```
Comment (kamillf) [inline CONTRIBUTING.md:76]: This line and the next--Ruby and Rake should both be capitalized. Not sure about bundler since I don't know what that is.

**Comment by kamillf** [comment]
There are other minor edits (like the ones I commented on) that should be made. But it would probably be easier if you merge it and let me do a PR on it (takes longer to write the comments)

A few larger comments: 
- Your headings are nouns, not verbs (well, technically gerunds). If I'm looking to do something, it's going to be harder for me to find in the document because the headings aren't about doing something. For example, instead of _Issues,_ you might call it _Submitting an Issue_ or _Submitting Feedback._ Similarly, _Pull Requests_ might be better as _Contributing Content._
- Would someone have Simpler without going through Github? I'm wondering if the steps about GitHub and git are needed. 
- I can't put my finger on it, but some of the organization seems maybe a bit off. My first thought in this regard was that maybe Issues should be elsewhere in the order. But I also wonder about IDEs and maybe Continuous Integration being more related to Pull Requests the current organization might suggest. Of course, not being a programmer, I don't entirely understand the Master and Tags section (i.e., why it's important) so that might influence my perspective. 
- Right now, the tone is a bit more laid back than the ReadMe. That could be fine, but it should be done as a conscious choice.

**Comment by gregoryjscott** [comment]
This was rolled into #21. Closing.

**Pull request closed without merging**